client/src/main/scala/com.crobox.clickhouse/stream/ClickhouseSink.scala

```diff
@@ -0,0 +1,33 @@
+package com.crobox.clickhouse.stream
+
+import akka.Done
+import akka.stream.scaladsl.{Flow, Keep, Sink}
+import com.crobox.clickhouse.ClickhouseClient
+import com.crobox.clickhouse.stream.ClickhouseBulkActor.{ClickhouseIndexingException, Insert}
+import com.typesafe.config.Config
+import com.typesafe.scalalogging.LazyLogging
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.concurrent.duration._
+
+object ClickhouseSink extends LazyLogging {
+
+  def insertSink(config: Config, client: ClickhouseClient)(implicit ec: ExecutionContext): Sink[Insert, Future[Done]] = {
```

Comment:
Maybe we should make the config configurable so that for different sinks you have different configs possible (1 with 1000 inserts batch other with 10000?